azure_functions_worker/bindings/shared_memory_manager.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import uuid
+from ..logging import logger
+from ..mmap_handler.file_writer import FileWriter
+from ..mmap_handler.file_reader import FileReader
+from ..mmap_handler.file_accessor import FileAccessor
+
+
+class SharedMemoryManager:
+    """
+    Performs all operations related to reading/writing data from/to Shared
+    Memory.
```

Comment:
As this is in the Python worker we could describe a little the exchange of data between C# and the worker.